tests/integration/sniffer/capturer.py

```
@@ -0,0 +1,858 @@
+def init_mongo_collection():
+    from pymongo import MongoClient
+    from pymongo.errors import CollectionInvalid
+
+    global mongo_collection
+    mongo_client = MongoClient(MONGO_URI)
+    mongo_db = mongo_client["integration-tests"]
+    # Ensure collection exists
+    try:
+        mongo_db.create_collection("packet_analysis")
+    except CollectionInvalid:
+        pass  # Collection already exists
```

Ruff (0.12.2)

Use `contextlib.suppress(CollectionInvalid)` instead of `try`-`except`-`pass`

(SIM105)